content/docs/tasks/security/health-check/index.md

```diff
@@ -80,13 +91,17 @@ Observe that the health checking interval is about 15 seconds, which is the defa
 
 ## (Optional) Configuring the health checking
 
-Optionally, adjust the health checking configuration to meet your own needs. Open the `istio.yaml` generated via [helm template](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template), and locate the following lines.
+Optionally, adjust the health checking configuration to meet your own needs. Open the file
+`install/kubernetes/istio-citadel-with-health-check.yaml`, and locate the following lines.
 
 {{< text plain >}}
 ...
   - --liveness-probe-path=/tmp/ca.liveness # path to the liveness health checking status file
   - --liveness-probe-interval=60s # interval for health checking file update
   - --probe-check-interval=15s    # interval for health status check
+  - --logtostderr
+  - --stderrthreshold
+  - INFO
```

Comment:
So I guess Citadel can't work well without these lines? If we need to add these lines, we should modify our yaml file template.

Reply:
To be consistent with the yaml file, these lines have been removed.